### DISCUSSION TITLE
AWS

### DISCUSSION THREAD

**Discussion by aambadkar** [learndevopsonline/learndevopsonline#1601] [Q&A] [answered]
Hello,
in my account before 1 week i terminated all created instance but today when i open i seen 1instance that automatically created and running from last week. they charge me 17.75 USD for that instance which i never created.

even today i terminated this instance but when i reload the page the instance is again automatically coming.

may i know if something is going wrong in my account

![image](https://github.com/learndevopsonline/learndevopsonline/assets/143367591/92f0a036-e6aa-44c5-9fb5-fcfc655fc554)
![image](https://github.com/learndevopsonline/learndevopsonline/assets/143367591/afcc30a9-2439-432e-b16f-99ad1c3a940a)
![image](https://github.com/learndevopsonline/learndevopsonline/assets/143367591/18ed7eb5-e024-4667-abdc-cfbdd896b0f7)

**Comment by sreedharm07** [comment, marked as answer]
Hi @aambadkar 
i think you created instance with spot request.
to delete that instance you should stop spot request
